frontend/src/pages/person/_itemId/index.vue

```diff
@@ -157,116 +124,96 @@ import { getBlurhash } from '@/utils/images';
-        sortOrder: [SortOrder.Descending],
-        recursive: true,
-        includeItemTypes: [BaseItemKind.Series],
-        fields: Object.values(ItemFields),
-        userId: remote.auth.currentUserId.value
-      })
-    ).data.Items;
+const movies =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Movie],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    const books = (
-      await remote.sdk.newUserApi(getItemsApi).getItems({
-        personIds: [itemId],
-        sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-        sortOrder: [SortOrder.Descending],
-        recursive: true,
-        includeItemTypes: [BaseItemKind.Book],
-        fields: Object.values(ItemFields),
-        userId: remote.auth.currentUserId.value
-      })
-    ).data.Items;
+const series =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Series],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    const photos = (
-      await remote.sdk.newUserApi(getItemsApi).getItems({
-        personIds: [itemId],
-        sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-        sortOrder: [SortOrder.Descending],
-        recursive: true,
-        includeItemTypes: [BaseItemKind.Photo],
-        fields: Object.values(ItemFields),
-        userId: remote.auth.currentUserId.value
-      })
-    ).data.Items;
+const books =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Book],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    let activeTab = 4;
+const photos =
+  (
+    await remote.sdk.newUserApi(getItemsApi).getItems({
+      personIds: [itemId],
+      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
+      sortOrder: [SortOrder.Descending],
+      recursive: true,
+      includeItemTypes: [BaseItemKind.Photo],
+      fields: Object.values(ItemFields),
+      userId: remote.auth.currentUserId.value
+    })
+  ).data.Items || [];
 
-    if (movies?.length) {
-      activeTab = 0;
-    } else if (series?.length) {
-      activeTab = 1;
-    } else if (books?.length) {
-      activeTab = 2;
-    } else if (photos?.length) {
-      activeTab = 3;
-    }
+const activeTab = ref(4);
```

Comment:
Why 4?

Reply:
Because before my refactor it was 4. I guess that's because there is always something to display in the informations tab :p